pages/vi/vi-nation.md

```diff
@@ -15,7 +15,10 @@ There should be constant communication between the nation and the communities. W
 Make sure vagrant is running and then click [here](http://localhost:3100) to access your Community Planet. 
 It is recommended that you use Firefox since Planet is only guaranteed to run properly on Firefox.
 
-**NOTE**: Before you can sync your community with the nation, you will need to create an additional dummy user on your community. Therefore, create one by clicking on "Become a Member" under the SIGN-IN button. Then fill out the details and press the "Become a Member" button. (Tip: When creating the dummy user, don't put a password that you actually use). Next, log back in to your admin account and double-check that your dummy account is listed under Members on the Manager page. Now that your community has a user, you can sync with the nation. If you can not create a dummy user, please see Q22 on the [FAQ page](vi-faq.md).
+**NOTE**: After you register your community, but before you can sync with the nation, you need to create an additional dummy user on your community. Therefore, to create a dummy user:
+* Create a quick additional user under "Become a Member" on the login page (HINT: When creating the dummy user, don't give it   a password that you actually use). 
```

Comment:
You need to add one line to separate as list. As there is not empty line it is taken as string not bullet.

Reply:
If you put empty line break it might be shown different than NOTE. So solution could be to put either bullet escape character or use ordered list instead of unordered list